akka-http-core/src/main/scala/akka/http/impl/engine/HttpConnectionTimeoutException.scala

```diff
@@ -0,0 +1,5 @@
+package akka.http.impl.engine
+
+import scala.util.control.NoStackTrace
+
+class HttpConnectionTimeoutException(msg: String) extends RuntimeException(msg) with NoStackTrace
```

Comment:
That was bady copy&paste programming :) TokenSourceActor.scala has no license header either